src/System.Drawing.Common/src/System/Drawing/Icon.cs

```diff
@@ -205,33 +150,29 @@ private static Icon ExtractAssociatedIcon(string filePath, int index)
             }
             if (uri.IsUnc)
             {
-                throw new ArgumentException(SR.Format(SR.InvalidArgument, "filePath", filePath));
+                throw new ArgumentException(SR.Format(SR.InvalidArgument, nameof(filePath), filePath));
             }
-            if (uri.IsFile)
+            if (!uri.IsFile)
```

Comment:
Not sure it's in the style guide, but we try to have a blank line after close curly, or another close curly or elseif, I think.